frame/identity/src/benchmarking.rs

```diff
@@ -153,7 +153,7 @@ benchmarks! {
 		};
 	}: _(RawOrigin::Signed(caller.clone()), Box::new(create_identity_info::<T>(x)))
 	verify {
-		assert_last_event::<T>(Event::<T>::IdentitySet(caller).into());
+		assert_last_event::<T>(Event::<T>::IdentitySet{who: caller}.into());
```

Comment:
```suggestion
		assert_last_event::<T>(Event::<T>::IdentitySet{ who: caller }.into());
```

Same for rest of file